backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/database/Schema.kt

```diff
@@ -51,6 +53,12 @@ object Cards : IntIdTable() {
                 ) or
                 ((activationSecretHash neq null) and (codeType eq CodeType.DYNAMIC))
         }
+        check("issuerid_or_entitlementsid_not_null") {
+            (
+                ((issuerId neq null) and (entitlementsId eq null))
+                    or ((issuerId eq null) and (entitlementsId neq null))
```

Comment:
:question: Am I correct in the assumption that for bavaria/nuernberg `issuerId` must not be null and for koblenz `entitlementsId` must not be null? Shouldn't we rather make the check take that into account (if that is even possible)?